tests/asserts/final/publication/geoserver_util.py

```diff
@@ -7,3 +11,9 @@ def is_complete_in_workspace_wms_instance(wms_instance, name, *, validate_metada
     if validate_metadata_url:
         assert len(wms_layer.metadataUrls) == 1
         assert wms_layer.metadataUrls[0]['url'].startswith('http://localhost:3080/record/xml/m-')
+
+
+def get_wms_layername(workspace, name):
```

Comment:
Do you plan also to add and use `get_wfs_layername` method?

Reply:
Well, it make sense if we keep `names` package with current functionality.